pywren/wren.py

```diff
@@ -164,6 +166,11 @@ def invoke_with_keys(self, s3_func_key, s3_data_key, s3_output_key,
         logger.info("call_async {} {} lambda invoke ".format(callset_id, call_id))
         lambda_invoke_time_start = time.time()
 
+        # overwrite args
+        # this is entirely for debugging various wrenhandler errors
```

Comment:
This is only used for testing - Is my reading right ? If so you can put that in the comment